### ISSUE TITLE
[Menu] Quick Menu entries: Add file path and file name to the pop-up notifications

### ISSUE THREAD

**Issue by davidhedlund** [libretro/RetroArch#16146]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

The directory path followed by file name is not displayed in the pop-up notifications.

### Expected behavior

The directory path followed by file name should be displayed in the pop-up notifications, so _easily_ can find them on your computer.  It's inconvenient to go to `Main Menu -> Directories` to find out where they are located.

### Actual behavior

[What is actually happening]

### Steps to reproduce the bug

* `Save States -> Save State`: "Saved state to slot #0."
* `Save States -> Load State`: "Loaded state to slot #0."
* `Core Options -> Manage Core Options -> Save Game Options`: "Core options file created successfully."
* `Core Options -> Manage Core Options -> Remove Game Options`: "Core options file removed successfully."
* `Core Options -> Manage Core Options -> Save Content Directory Options`:  "Core options file created successfully."
* `Core Options -> Manage Core Options -> Save Content Directory Options`: "Core options file removed successfully."
* `Controls -> Manage Remap Files -> Save Remap File As`: "Remap file saved successfully.". The file name is unique, the virtual keyboard pop-ups and you have to define the file name yourself.
* `Controls -> Manage Remap Files -> Save Game Remap File`: "Remap file saved successfully."
* `Controls -> Manage Remap Files -> Remove Game Remap File`: "Remap file removed successfully."
* `Controls -> Manage Remap Files -> Save Content Directory Remap File`: "Remap file saved successfully."
* `Controls -> Manage Remap Files -> Remove Content Directory Remap File`: "Remap file removed successfully."
* `Controls -> Manage Remap Files -> Save Core Remap File`: "Remap file saved successfully."
* `Controls -> Manage Remap Files -> Remove Core Remap File`: "Remap file removed successfully."

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.16.0

### Environment information

- OS: Any
- Compiler: [In case you are running local builds]

**Comment by hizzlekizzle** [comment]
>     Main Menu -> Online Updater -> Updater Settings -> On-Demand Thumbnail Downloads: "Automatically download missing thumbnails while browsing playlists" <-- s/playlists/the Playlists directory/
>     Main Menu -> Online Updater -> Information -> Database Manager: "View databases." <-- s/View databases/View files in the Database directory/

I disagree with these. In neither case are you browsing the directory. You're browsing the contents of the files.

While I like the idea of including the full paths, there's no way most full paths are going to fit in the notification widgets. I suggest including the full path in the logs if they're not already, though.

**Comment by davidhedlund** [comment]
> > ```
> > Main Menu -> Online Updater -> Updater Settings -> On-Demand Thumbnail Downloads: "Automatically download missing thumbnails while browsing playlists" <-- s/playlists/the Playlists directory/
> > Main Menu -> Online Updater -> Information -> Database Manager: "View databases." <-- s/View databases/View files in the Database directory/
> > ```
> 
> I disagree with these. In neither case are you browsing the directory. You're browsing the contents of the files.

I removed them, thank you for reading through the whole list!

Do you prefer single quotes or not?:
* `Change 'File Browser' settings.` or
* `Change File Browser settings.`

> While I like the idea of including the full paths, there's no way most full paths are going to fit in the notification widgets.

I was actually going to submit an issue about that right now. Here you go:
* https://github.com/libretro/RetroArch/issues/16147

**Comment by davidhedlund** [comment]
@zoltanvb Please participate in this issue if you want.

**Comment by hizzlekizzle** [comment]
I'd probably go with no quotes, since that sort of thing can be difficult to convey in translations.

**Comment by davidhedlund** [comment]
> While I like the idea of including the full paths, there's no way most full paths are going to fit in the notification widgets. I suggest including the full path in the logs if they're not already, though.

I just submitted this, if you are interested:
* https://github.com/libretro/RetroArch/issues/16222

**Comment by davidhedlund** [comment]
@zoltanvb I updated all entries in the top post:
* Made all text significantly more readable.
* Merged fixed entries under the "Done" section

**Comment by zoltanvb** [comment]
Prepend sublabel was fixed by #16095.
Resampler directory seems unused nowadays, can probably be removed from the code.
Video font directory gets initialized when browsing for a new font so probably it is not needed to have it configurable separately.
Core options path is only used when global core options are enabled (and in that case it points to one file called retroarch-core-options.cfg).
Cursor directory is not used any more (views are saved next to playlists), few traces are still in the code that could be removed.
Bundle_assets dirs are used, but that should probably not be user configurable.
DualAnalog is coming from the core(s) directly, so that does not depend on RA,

**Comment by davidhedlund** [comment]
> @zoltanvb I updated all entries in the top post:
> 
>     * Made all text significantly more readable.
> 
>     * Merged fixed entries under the "Done" section

I repeated this with your new information. Thank you once again.

**Comment by davidhedlund** [comment]
These are all entries in "Settings -> Drivers". Can someone please answer which one that should have the sub-label "Requires restart.", because I don't know how to test them?:

* Menu: Has "Requires restart."
* Video: Yes/no?
* Audio: Yes/no?
* Microphone: Yes/no?
* Input: Yes/no?
* Controller: Yes -- Confirmed by me
* Record: Yes/no?
* MIDI: Yes/no?
* Camera: Yes/no?
* Location: Yes/no?

**Comment by davidhedlund** [comment]
Found that "Maximum Users" requires restart to functioning, which should be added to its sub-label:

Settings -> Input -> Maximum Users
Sub-label: "[Current text ] (Restart required)"

**Comment by davidhedlund** [comment]
* https://github.com/libretro/RetroArch/issues/18802
